examples/examples-cpu/nyc-taxi-snowflake/hyperparameter-dask.ipynb

```diff
@@ -269,7 +269,10 @@
     "pipeline = Pipeline(steps=[\n",
```

Comment:
<p>oh nice! I did not know about this before. <a href="https://scikit-learn.org/stable/modules/generated/sklearn.compose.ColumnTransformer.html#sklearn-compose-columntransformer" target="_blank">https://scikit-learn.org/stable/modules/generated/sklearn.compose.ColumnTransformer.html#sklearn-compose-columntransformer</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>